package.json

```diff
@@ -29,6 +29,8 @@
     "d3-zoom": "^1.7.3",
     "echarts": "^4.2.1",
     "echarts-for-react": "^2.0.14",
+    "enzyme": "^3.2.0",
```

Comment:
Is enzyme needed in production?

Reply:
Great q, my linter was complaining at me that enzyme was in `devDepdendencies` and not dependencies so I moved it here to keep it happy. I was actually surprised to learn this but it turns out that when we're using webpack, it doesn't matter if a dep is in `devDependencies` or `dependencies`, it only gets included in production if a file imports it. So I believe this has no impact on the production bundle as long as no non-test files import enzyme.